packages/cozy-stack-client/src/JobCollection.js

```diff
@@ -32,11 +45,42 @@ class JobCollection {
     })
   }
 
+  /**
+   * Return a normalized job, given its id
+   */
   async get(id) {
     return Collection.get(this.stackClient, uri`/jobs/${id}`, {
       normalize: normalizeJob
     })
   }
+
+  /**
+   * Polls a job state until it is finished
+   *
+   * `options.until` can be used to tweak when to stop waiting. It will be
+   * given the current job state. If true is returned, the awaiting is
+   * stopped.
+   */
+  async waitFor(id, options = {}) {
+    options.until = options.until || hasJobFinished
+    options.delay = options.delay === undefined ? 5 * 1000 : options.delay
+    options.timeout =
+      options.timeout === undefined ? 60 * 5 * 1000 : options.timeout
```

Comment:
why not defining the default value in the declaration ? I find it clearer to read and understand 

```js
async waitFor(id, options = {
timeout: 60 * 5 * 1000,
delay: 5 * 1000
}) {
```

Reply:
ca n'a pas le meme comportement si tu passes un objet partiel d'options.

avec ton implementation, si tu fais:

```
waitFor(id, { delay: 3 * 1000 })
```

tu perds le timeout par défaut.

Reply:
<img width="393" alt="Capture d’écran 2019-10-04 à 14 36 24" src="https://user-images.githubusercontent.com/1107936/66207963-5f5ddf00-e6b4-11e9-84e3-39fca2b01e9a.png">

🤔 . J'ai mal compris un truc ?

Reply:
```
> fn = (opts = { a: 1, b: 2}) => { console.log(opts.a, opts.b) }
[Function: fn]
> fn()
1 2
undefined
> fn({a:0})
0 undefined // 😱 no b !
```

Reply:
Fixed by using the destructing method ({ arg = defaultValue }). Thanks